### ISSUE TITLE
Searchbox keyboardnavigatable error

### ISSUE THREAD

**Issue by SychO9** [flarum/core#3210]
## Bug Report

**Current Behavior**
When setting the forum to members only, visitors see a broken homepage, and searchbox is missing.

```
Cannot read properties of undefined (reading 'addEventListener')
    at t.e.bindTo (KeyboardNavigatable.ts:141)
    at e.n.oncreate (Search.tsx:207)
```

**Steps to Reproduce**
1. Set forum access to members only through permissions.
2. Logout
3. View broken page

**Comment by askvortsov1** [comment]
I believe this was fixed in #3228?

**Comment by SychO9** [comment]
Oh hadn't noticed, perfect!
